services/slam/fake/data_loader.go

```diff
@@ -113,6 +113,42 @@ func fakeGetInternalState(datasetDir string, slamSvc *SLAM) ([]byte, error) {
 	return data, nil
 }
 
+func fakeGetPointCloudMapStream(datasetDir string, slamSvc *SLAM) (func() ([]byte, error), error) {
+	path := filepath.Clean(artifact.MustPath(fmt.Sprintf(pcdTemplate, datasetDir, slamSvc.getCount())))
+	slamSvc.logger.Debug("Reading " + path)
+	file, err := os.Open(path)
+	if err != nil {
+		return nil, err
+	}
+	f := func() ([]byte, error) {
+		chunk := make([]byte, 64*1024)
```

Comment:
Can be placed outside of func to avoid redefinition (see client_test.go)

Reply:
Note: send over extra data shouldn't be an issue as we only send over the written bytes